docs/administration/scim.md

```diff
@@ -47,38 +47,42 @@ Fill out the fields as shown in the screenshot above: Paste the base URL from th
 * For Unique identifier field for users, set to "email". 
 * Check the Push New Users and Push Profile Updates. 
 * For Authentication Mode, select HTTP Header and paste the Bearer Token from the WorkOS set up to the field. 
-* Click Save.  
-
-### Attributes and Roles on Okta
-
-To setup Attribute Mapping, you will see that we support a custom attribute called `eppoMemberRole`. This optional custom attribute allows an IT admin to set the user's Eppo role from within the IdP.
-
-That attribute can only have the following attributes (strings): `default`, `viewer`, `experiment_editor`, `data_owner`, `admin`
-
-The `default` value is useful for migrating to managing roles in the Idp: it will keep the user's role as it is in Eppo, or if the user is new, it will assume the default user role as configured in Eppo.
-
-<img src="/img/administration/scim/scim-custom-attribute.png" alt="configure Okta API Integration" width="600" />
+* Click Save. 
+
+To assign users and groups:
+* Navigate to the assignments tab.
+* Click the Assign button and select Assign to Groups.
+* Select the groups you want to assign to the application.
+* Click Save.
+
+To push groups for role assignment: (see User roles via Groups below for more details about roles)
```

Comment:
```suggestion
To push groups for role assignment: (see [User roles via Groups](#user-roles-via-groups) below for more details about roles)
```

Should we add something like this (can't remember exactly how it works) so clicking on this text takes them straight to the section?